scripts/ratchet_coverage/install_cargo_llvm_cov.py

```
@@ -4,16 +4,19 @@
 
 
 def main() -> None:
     try:
         cargo["install", "cargo-llvm-cov"]()
         typer.echo("cargo-llvm-cov installed successfully")
-    except Exception as e:
-        typer.echo(f"Failed to install cargo-llvm-cov: {e}", err=True)
-        raise typer.Exit(code=1)
+    except ProcessExecutionError as exc:
+        typer.echo(
+            f"cargo install failed with code {exc.retcode}: {exc.stderr}", err=True
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)